PennMobile/Dining/SwiftUI/DiningViewModelSwiftUI.swift

```diff
@@ -21,9 +21,9 @@ class DiningViewModelSwiftUI: ObservableObject {
 
     @Published var alertType: NetworkingError?
 
-    @Published var swipes = 0
-    @Published var diningDollars = 0.0
-    @Published var guestSwipes = 0
+    @Published var swipes = UserDefaults.standard.getdiningBalance()?.regularVisits ?? 0
+    @Published var diningDollars = Double(UserDefaults.standard.getdiningBalance()?.diningDollars ?? "0.0")!
+    @Published var guestSwipes = UserDefaults.standard.getdiningBalance()?.guestVisits ?? 0
```

Comment:
Instead of having three variables each breaking up parts of diningBalance, we should simply have a `@Published var diningBalance: DiningBalance`, initialized with all zeros if it doesn't exist